app/src/main/java/com/keylesspalace/tusky/util/CustomEmojiHelper.kt

```diff
@@ -45,7 +46,9 @@ fun CharSequence.emojify(emojis: List<Emoji>?, view: View) : CharSequence {
     if(emojis.isNullOrEmpty())
         return this
 
-    val builder = SpannableStringBuilder.valueOf(this)
+    val builder = SpannableString.valueOf(this)
```

Comment:
Why this change?

Reply:
Oh, that seems to be a part of older research when I tried to make this custom emojis to not ANR. 

There is no need in SSB, when we're not modifying the text, so we can use just SpannableString.

Despite this change doesn't seem to fix #1959 and the core issue was text line breaking, I still think using SS rather than SSB is pretty harmless. The only place where SSB must be used is LinkHelper [here](https://github.com/tuskyapp/Tusky/blob/develop/app/src/main/java/com/keylesspalace/tusky/util/LinkHelper.java#L127).

Reply:
The reason why SSB was used here is because when the emojify method is used together with the one in LinkHelper, `valueOf` saves one instance creation because an existing one is reused. Not sure if it has a real impact though.

Reply:
That makes sense. Gonna revert it then.

Still I think using SSB is quite overhead and even LinkHelper should not use it but that's for another discussion.